### PR TITLE
First noise

### PR DESCRIPTION
**Pull request by olenkajain**

### CODE REVIEW

File: pax/simulation.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 import time
 import pickle
 from functools import partial
-
+from scipy.fftpack import irfft, rfft,rfftfreq
```
Comment (tunnell) [inline pax/simulation.py]: __[flake8]__
* [E231] missing whitespace after ','

File: pax/simulation.py [hunk 2]
```diff
@@ -315,6 +316,8 @@ def make_pax_event(self):
             noise_sample_len = self.config['real_noise_sample_size']
             available_noise_samples = self.noise_data.shape[1] // noise_sample_len
             needed_noise_samples = int(math.ceil(pulse_length / noise_sample_len))
+            choice=np.random.normal(self.noise_data,.05*sqrt(abs(self.noise_data)))
```
Comment (tunnell) [inline pax/simulation.py]: __[flake8]__
* [E225] missing whitespace around operator
* [E231] missing whitespace after ','

File: pax/simulation.py [hunk 3]
```diff
@@ -315,6 +316,8 @@ def make_pax_event(self):
...
+            noise_to_add=irfft(choice)
```
Comment (tunnell) [inline pax/simulation.py]: __[flake8]__
* [E225] missing whitespace around operator

File: pax/simulation.py [hunk 4]
```diff
@@ -465,33 +468,31 @@ def make_pax_event(self):
 
             # Did you want to superpose onto real noise samples?
             if self.config['real_noise_file']:
-                if noise_sample_mode != 'coherent':
+                #if noise_sample_mode != 'coherent':
```
Comment (tunnell) [inline pax/simulation.py]: __[flake8]__
* [E265] block comment should start with '# '

File: pax/simulation.py [hunk 5]
```diff
@@ -465,33 +468,31 @@ def make_pax_event(self):
...
                     # For each channel, choose different noise sample numbers
-                    chosen_noise_sample_numbers = np.random.randint(0,
+                    #chosen_noise_sample_numbers = np.random.randint(0,
```
Comment (tunnell) [inline pax/simulation.py]: __[flake8]__
* [E265] block comment should start with '# '

File: pax/simulation.py [hunk 6]
```diff
@@ -465,33 +468,31 @@ def make_pax_event(self):
...
                                                                     available_noise_samples - 1,
                                                                     needed_noise_samples)
 
-                    roll_number = np.random.randint(noise_sample_len)
+                    #roll_number = np.random.randint(noise_sample_len)
```
Comment (tunnell) [inline pax/simulation.py]: __[flake8]__
* [E128] continuation line under-indented for visual indent
* [E265] block comment should start with '# '

File: pax/simulation.py [hunk 7]
```diff
@@ -465,33 +468,31 @@ def make_pax_event(self):
...
 
                 # Extract the chosen noise samples and concatenate them
                 # Have to use a listcomp here, unless you know a way to select multiple slices in numpy?
                 #  -- yeah making an index list with np.arange would work, but honestly??
-                real_noise = np.concatenate([
-                    self.noise_data[channel - self.channel_offset][nsn * noise_sample_len:(nsn + 1) * noise_sample_len]
-                    for nsn in chosen_noise_sample_numbers
-                ])
+
+                real_noise = noise_to_add[channel-self.channel_offset]
```
Comment (tunnell) [inline pax/simulation.py]: __[flake8]__
* [E128] continuation line under-indented for visual indent
* [E251] unexpected spaces around keyword / parameter equals

File: pax/simulation.py [hunk 8]
```diff
@@ -465,33 +468,31 @@ def make_pax_event(self):
...
 
                 # Roll the noise samples by a fraction of the sample size,
                 # to avoid same artifacts falling at the same point every time
-                np.roll(real_noise, roll_number, axis=0)
+                #np.roll(real_noise, roll_number, axis=0)
```
Comment (tunnell) [inline pax/simulation.py]: __[flake8]__
* [E122] continuation line missing indentation or outdented
* [E265] block comment should start with '# '

File: pax/simulation.py [hunk 9]
```diff
@@ -465,33 +468,31 @@ def make_pax_event(self):
...
 
                 # Adjust the noise amplitude if needed, then add it to the ADC wave
                 noise_amplitude = self.config.get('adjust_noise_amplitude', {}).get(str(channel), 1)
                 if noise_amplitude != 1:
                     # Determine a rough baseline for the noise, then adjust towards it
                     baseline = np.mean(real_noise[:min(len(real_noise), 50)])
                     real_noise = baseline + noise_amplitude * (real_noise - baseline)
-                adc_wave += real_noise[:pulse_length]
+                adc_wave += real_noise[start_index:pulse_length+start_index]
```
Comment (tunnell) [inline pax/simulation.py]: __[flake8]__
* [E122] continuation line missing indentation or outdented

**Comment by olenkajain** [comment]
I’m sorry, this was accidental pull request!

**Pull request closed without merging**